test/unit/io_managers_test.py

```diff
@@ -218,6 +218,7 @@ def test_error_when_handling_view_without_metadata(pudl_sqlite_io_manager_fixtur
         pudl_sqlite_io_manager_fixture.handle_output(output_context, sql_stmt)
 
 
+@pytest.mark.xfail(reason="SQLAlchemy is not finding the view. Debug or remove.")
```

Comment:
I'm inclined to leave this `xfail()` for now, and see if it's easy to fix when integrating SQLAlchemy 2.0 (which we should do as soon as pandas 2.0 is merged in)

Reply:
I agree. We're not using any SQL views right now, but if we do want to use them at some point we need to make them work with SQLAlchemy 2.0, so I think in that PR we need to either figure out how to make this work or explicitly decide we're not going to use views (which is what we've de facto chosen to do thus far, given the complexities around defining their metadata correctly)